src/libs/state.ts

```diff
@@ -108,37 +67,67 @@ export type AuthState = {
   isTimeoutEnabled?: boolean | undefined;
   nihStatus?: NihStatus;
   nihStatusLoaded: boolean;
-  profile: TerraUserProfile;
-  refreshTokenMetadata: TokenMetadata;
-  sessionId?: string | undefined;
-  sessionStartTime: number;
-  system: SystemProperties;
   termsOfService: TermsOfServiceStatus;
-  terraUser: TerraUser;
   terraUserAllowances: SamUserAllowances;
-  terraUserAttributes: SamUserAttributes;
-};
-
-export type FenceStatus = {
-  [key: string]: BondFenceStatusResponse;
-};
+}
 
 export const authStore: Atom<AuthState> = atom<AuthState>({
-  anonymousId: undefined,
-  authTokenMetadata: {
-    token: undefined,
-    id: undefined,
-    createdAt: -1,
-    expiresAt: -1,
-    totalTokenLoadAttemptsThisSession: 0,
-    totalTokensUsedThisSession: 0,
-  },
   cookiesAccepted: undefined,
   fenceStatus: {},
   hasGcpBillingScopeThroughB2C: false,
   signInStatus: 'uninitialized',
   userJustSignedIn: false,
   nihStatusLoaded: false,
+  termsOfService: {
+    permitsSystemUsage: undefined,
+    isCurrentVersion: undefined,
+  },
+  terraUserAllowances: {
+    allowed: false,
+    details: {
+      enabled: false,
+      termsOfService: false,
+    },
+  },
+});
+
+export interface TerraUser {
+  token?: string | undefined;
+  scope?: string | undefined;
+  id?: string | undefined;
+  email?: string | undefined;
+  name?: string | undefined;
+  givenName?: string | undefined;
+  familyName?: string | undefined;
+  imageUrl?: string | undefined;
+  idp?: string | undefined;
+}
+
+export interface TerraUserProfile {
+  // TODO: anonymousGroup is here from getProfile from orch
+  // TODO: for future ticket, separate items updated via register/profile (personal info)
+  //  from things that are updated via api/profile/preferences (starred workspaces, notification settings)
+  firstName: string | undefined;
```

Comment:
Would it be possible to standardize the name… names? I know they come from different user objects but bugs me that we have `name`, `givenName`, `familyName`, then also `firstName` and `lastName` on various user objects…